flexmeasures/cli/data_add.py

```diff
@@ -1336,14 +1342,20 @@ def add_schedule_process(
         }
 
     if as_job:
-        job = create_scheduling_job(sensor=power_sensor, **scheduling_kwargs)
+        job = create_scheduling_job(
+            asset_or_sensor_id=power_sensor, **scheduling_kwargs
```

Comment:
The `create_scheduling_job` function asks for `asset_or_sensor` rather than `asset_or_sensor_id`.